test/unit/test_quadlet.py

```diff
@@ -48,7 +50,8 @@ def test_quadlet_generate(input: Input, expected_files_path: Path):
         with io.StringIO() as sio:
             file.config.write(sio)
             sio.seek(0)  # rewind
-
+            print(sio.read())
```

Comment:
Is this debug or expected?

Reply:
Ups, forgot to remove this. Used it for getting the fixtures. Removed.